ecommerce/aws/templates/edx/partials/_base_navbar.html

```diff
@@ -39,12 +39,12 @@
               {% block small_dropdown_menu %}{% endblock small_dropdown_menu %}
           </li>
         {% else %}
-          {% if custom_settings.COUPONS_ENABLE_LOGIN_BUTTON|default_if_none:True or request.path|slice:":15" != "/coupons/offer/" %}
+          {% if not custom_settings.COUPONS_DISABLE_LOGIN_BUTTON|default_if_none:False or not request or not request.path or "/coupons/offer/" not in request.path|default_if_none:"" %}
```

Comment:
`{% if not custom_settings.COUPONS_DISABLE_LOGIN_BUTTON|default_if_none:False or not request or not request.path or not "/coupons/offer/" in request.path|default_if_none:"" %}`
Is this possible to maintain consistency? In addition, please add an inline-comment about this, so it can be easily figured it out. @01001110J